mono/metadata/w32socket-unity.c

```diff
@@ -3,15 +3,26 @@
 
 #if defined(PLATFORM_UNITY) && defined(UNITY_USE_PLATFORM_STUBS)
 
+#ifdef NO_HAVE_TRANSMIT_FILE_BUFFERS
+
+typedef struct {
+	gpointer Head;
+	guint32 HeadLength;
+	gpointer Tail;
+	guint32 TailLength;
+} TRANSMIT_FILE_BUFFERS;
+
+#endif
```

Comment:
Note that I've just had to add this one one of our platforms. It is a bit of a hack, but I think it is acceptable for now, as it gets the build going, and it is only used in a stub method that won't be called anywhere in the debugger code.